src/HeaderBar/index.js

```diff
@@ -34,27 +34,35 @@ export const HeaderBar = ({ appName, className }) => {
         },
     })
 
-    if (loading) return <span>...</span>
-
     if (error) return <span>{`ERROR: ${error.message}`}</span>
 
-    const locale = data.user.settings.keyUiLocale || 'en'
-    i18n.changeLanguage(locale)
+    if (!loading) {
+        // TODO: This will run every render which is probably wrong!  Also, setting the global locale shouldn't be done in the headerbar
```

Comment:
This shouldn't be in headerbar I don't think....

Reply:
Agreed. I forgot to remove this after implementing i18n. It should be moved to the Storybook part since it simulates the App role.

Reply:
Should we do that in another place then?

Reply:
Yeah, just leave it as it is with the `TODO` and we can do it in a different PR.